docs/articles/user-guide/development-models.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Development Models with VS Code
+---
+
+## Overview
+
+Salesforce Extensions for VS Code supports Org Development and Package Development models to authorize, create and switch orgs in your project:
+
+- **Org Development Model**: Allows you work with orgs that don’t have source tracking, such as sandbox, Developer Edition (DE) org, Trailhead Playground, or even a production org to retrieve and deploy code directly.
+- **Package Development Model**: Allows you to create self-contained applications or libraries that are deployed to your org as a single package. These packages are typically developed against source-tracked orgs such as scratch orgs. This development model uses org source tracking, source control, and continuous integration and deployment.
+
+Salesforce Extensions for VS Code runs commands against the org that you’ve set as your default org for development.
+
+## Org Development Model
+
+Use the Org Development model work with orgs that don’t have source tracking, such as sandboxes, Developer Edition (DE) orgs, or Trailhead Playgrounds, in Visual Studio Code. With this development model, you must track changes manually and deploy sets of changes to sandboxes and then to your production org. See the [Org Development Model](https://trailhead.salesforce.com/content/learn/modules/org-development-model) Trailhead module.
+
+![Demo](../../images/changeset-demo.gif)
+
+To start developing with this model:
+
+- Create a project.
+- Use the Org Picker to authorize an org you want develop against.
+- Use Manifest or Org Browser to retrieve source from the default org.
+- Deploy source to the default org.
+
+## Create Project with Manifest
+
+To create developing against non-source-tracked orgs:
+
+1. Open the VS Code editor and from the Command Palette, run **SFDX: Create Project with Manifest**.
```

Comment:
From a new user's perspective, I would want these steps included for bringing up the Command Palette:
(press Ctrl+Shift+P on Windows or Linux, or Cmd+Shift+P on macOS)